book/03-hacking-atom/sections/06-iconography.asc

```diff
@@ -0,0 +1,27 @@
+=== Iconography
+
+Every so often you come to a point in time, where, to represent something, an icon is better off than words are.
+Since Atom is made by GitHub, their https://octicons.github.com/[Octicons] font is available for use.
```

Comment:
> icon is better off than words

Some argue that "[The best icon is a text label](http://thomasbyttebier.be/blog/the-best-icon-is-a-text-label)". So maybe we should encourage people to use text labels if possible and only if space gets tight fall back to an icon. Or often icons get used in addition to a text label.

> Since Atom is made by GitHub

Not sure about mentioning this as the main reason. I guess it is, but just sounds a bit strange in this context. How about that whole first paragraph simplified to something like:

> Atom comes bundled with the [Octicons](https://octicons.github.com/). You can use these icons in your packages if space for a text label is insufficient or to add a visual clue to an existing label.